custom-connectors/PUGGamifiedEngagement/apiDefinition.swagger.json

```diff
@@ -0,0 +1,681 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "PICNIC CONNECTOR",
+    "description": "PICNIC CONNECTOR",
```

Comment:
@smart-jp Same here, The description value should be more explanative of your connector.

Documentation - [Description](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)